backend/app/features/operator_history/__init__.py

```
@@ -0,0 +1,17 @@
+    OperatorHistoryResultSummary,
+    OperatorHistoryRunSummary,
+)
+
+__all__ = [
+    "OperatorHistoryRequestSummary",
+    "OperatorHistoryCandidateSummary",
+    "OperatorHistoryRunSummary",
+    "OperatorHistoryDenialSummary",
+    "OperatorHistoryInvalidationSummary",
+    "OperatorHistoryResultSummary",
+]
```

Ruff (0.15.10)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)